modules/es-extensions/store/extensions/app/greg-store-defaults/themes/store/partials/navigation.hbs

```diff
@@ -55,6 +55,7 @@
                         </div>
                     {{/if}}
                     {{> categorization-entry .}}
+                    {{#if assetCategoryFilterDetails}}<style>div.body-container{min-height: 1220px;}</style>{{/if}}
```

Comment:
Better to move this internal CSS to an external file. We do not recommend to have styles inside body area.